packages/@okta/vuepress-site/docs/reference/rate-limits/index.md

```diff
@@ -45,6 +45,6 @@ If an org-wide rate limit is exceeded, an HTTP 429 status code is returned. You
 
 * [Previous rate limits](/docs/reference/rl-previous/): This content covers the rate limits for orgs that were created before 2018-05-17.
 
-* [Okta API endpoints and per-user limits](/docs/reference/rl-additional-limits/#okta-api-endpoints-and-per-user-limits): API endpoints that take username and password credentials, including the [Authentication API](/docs/reference/api/authn/) and the [OAuth 2.0 resource owner password flow](/docs/guides/implement-password/), have a per-username rate limit to prevent brute force attacks with the user's password.
+* [Okta API endpoints and per-user limits](/docs/reference/rl-additional-limits/#okta-api-endpoints-and-per-user-limits): API endpoints that take username and password credentials, including the [Authentication API](/docs/reference/api/authn/), the [OAuth 2.0 resource owner password flow](/docs/guides/implement-password/), and [SMS and Call factor endpoints](/docs/reference/api/factors/#factor-lifecycle-operations), have a per-username rate limit to prevent brute force attacks with the user's password.
```

Comment:
SMS and Call don't "take username and password" so maybe they don't belong in the "including" list?  The heading includes per-user limits, which it is - so probably the right section.  It's just the example says it is listing username/password APIs, which it isn't.